astTraversal/type.go

```diff
@@ -205,21 +205,15 @@ func (t *TypeTraverser) Result() (Result, error) {
 		fields := make(map[string]Result)
 		for i := 0; i < n.NumFields(); i++ {
 			f := n.Field(i)
+			name := f.Id()
+			isExported := f.Exported()
```

Comment:
Is this the correct field to use here? I thought that it indicated something else.

The reason I checked the is exported the other time was after we did the tag validation

Reply:
It might not be, but it seems to be working, what do you think it did?

I decided to check it's exported because if it isn't then you can do struct marshalling or un-marshalling so it can't be used as a type.

Reply:
If it works it works. Need to think about how we deal with custom marshalling at some point

Reply:
Agreed, don't think it should be too difficult to do?